include/zephyr/drivers/sensor/grow_r502a.h

```diff
@@ -71,6 +124,12 @@ enum sensor_attribute_grow_r502a {
 	 *	val->val2	matching score.
 	 */
 	SENSOR_ATTR_R502A_COMPARE,
+	/** To control device LED */
+	/**
+	 * @param val->val1	use R502A_LED_PARAMS macro with desired
+	 *			LED configurations.
+	 */
+	SENSOR_ATTR_R502A_DEVICE_LED,
```

Comment:
Please look at enabling the LED driver API

Reply:
LED is internal to the Fingerprint sensor and is accessible to indicate fps actions alone. I don't think we need this LED-controlling functionality to be part of the LED driver. please help me understand why and how? 
  
![image](https://github.com/zephyrproject-rtos/zephyr/assets/37179801/1742cb59-aabc-418f-8a1e-db1460ac94e0)

Reply:
@MaureenHelm ping

Reply:
@MaureenHelm ping

Reply:
It looks like you're enabling functionality similar to the LED API (on, off, blink, breathe). Rather than do it through a custom sensor attribute, I'm suggesting that you implement the LED API for this device

Reply:
@MaureenHelm Since this grow_r502a Fingerprint sensor comes under the sensor subsystem and LED is one of the attributes of this grow_r502a sensor, there won't be any need to control this LED as a separate entity, I think it would be best to control this FPS LED with SENSOR_ATTR alone.
If we implement LED API, does it not create any conflict of one device 2 driver API set?

Reply:
@MaureenHelm ping

Reply:
> @MaureenHelm Since this grow_r502a Fingerprint sensor comes under the sensor subsystem and LED is one of the attributes of this grow_r502a sensor, there won't be any need to control this LED as a separate entity, I think it would be best to control this FPS LED with SENSOR_ATTR alone. If we implement LED API, does it not create any conflict of one device 2 driver API set?

@MaureenHelm ping

Reply:
This can be somewhat resolved by having a single device's data/config structs be shared between two struct device's implementing the two APIs perhaps... but right its a bit confusing how this is actually supposed to work I agree. I think @bjarki-trackunit implemented multiple device APIs for a device before maybe he has some comments on this.